src/utils/GAUtils.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * 從 GA SDK 取得 User Pseudo ID，又稱 Client ID
+ * 是使用者在該 GA 資源、該 device 上的 user id
+ * Ref:
+ *  1. https://support.google.com/analytics/answer/12675187?hl=en
+ *  2. https://developers.google.com/tag-platform/gtagjs/reference#get
+ * @param {*} ga_measurement_id
+ * @returns
+ */
+export const getUserPseudoId = async ga_measurement_id => {
```

Comment:
return promise的話應該就不用 async

```suggestion
export const getUserPseudoId = ga_measurement_id => {
```